sylt/src/lib.rs

```diff
@@ -211,17 +271,29 @@ mod lua {
                 let mut file = file;
                 file.replace_range(file.rfind(".").unwrap().., ".lua");
 
-                let output = Command::new("lua")
-                    .arg(file)
-                    .stdin(Stdio::null())
-                    .stderr(Stdio::piped())
-                    .stdout(Stdio::null())
-                    .output()
-                    .expect(concat!("Failed to run ", $path));
+                let output = child.wait_with_output().unwrap();
+                // NOTE(ed): Status is always 0 - atleast on my version of lua - so we check
+                // for stderr - which is kinda a bad idea.
```

Comment:
This should be investigated more. I get non-zero statuses in my terminal at least.

Reply:
I, too, get a non-zero status.

```
$ lua not-lua.lua
lua: not-lua.lua:1: syntax error near ';'
$ echo $status
1
$ cat not-lua.lua
aflasjga;lsgj
```

Reply:
```
Lua 5.4.3  Copyright (C) 1994-2021 Lua.org, PUC-Rio
```

Reply:
It's ONLY when you pipe the file - I might not have been clear about that:
```
$ echo "not-lua&&%¤&%¤%" | lua
```